mediarouter/mediarouter/src/main/java/androidx/mediarouter/media/MediaRouteProviderProtocol.java

```diff
@@ -164,7 +164,7 @@ abstract class MediaRouteProviderProtocol {
     public static final String CLIENT_DATA_MEMBER_ROUTE_ID = "memberRouteId";
     public static final String CLIENT_DATA_ROUTE_CONTROLLER_OPTIONS = "routeControllerOptions";
 
-    public static final String DATA_KEY_GROUPABLE_SECION_TITLE = "groupableTitle";
+    public static final String DATA_KEY_GROUPABLE_SECTION_TITLE = "groupableTitle";
```

Comment:
The containing class is a package-private class. So this shouldn't be a breaking change.